rest-e2e-tests/streamr-api-clients.js

```diff
@@ -240,7 +240,7 @@ class Streams {
             .methodAndPath('POST', `streams/${id}/permissions`)
```

Comment:
To make a stream public, need to grant (with `anonymous: true`) both `stream_get` and `stream_subscribe`